src/VscPreviewExtension/Microsoft.DocAsCode.Dfm.VscPreview/DocfxProcess.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Dfm.VscPreview
+{
+    using System;
+    using System.IO;
+    using CsQuery;
+
+    using Microsoft.DocAsCode.Build.Engine;
+    using Microsoft.DocAsCode.Plugins;
+
+    public class DocfxProcess : DocFxPreviewProcess
```

Comment:
Yes, use Docfx later

---
In reply to: [101437646](https://github.com/dotnet/docfx/pull/1313#discussion_r101437646) [](ancestors = 101437646)